main.py

```diff
@@ -286,10 +300,10 @@ def run_beast2(simulation_xml):
         $ ./lib/beast/bin/beast -seed 1 -overwrite <simulation_xml>
         """
         print(f"Running simulation: {simulation_xml}")
-        command = ["./lib/beast/bin/beast", "-seed", "1", "-overwrite", simulation_xml]
+        command = ["/Applications/BEAST 2.7.6/bin/beast", "-seed", "1", "-overwrite", simulation_xml]
```

Comment:
The following is is a better way to do this:

```
beast_executable_mac = "/Applications/BEAST 2.7.6/bin/beast"
beast_executable_linux = "./lib/beast/bin/beast"
if os.path.exists(beast_executable_mac):
    beast_executable = beast_executable_mac
elif os.path.exists(beast_executable_linux):
    beast_executable = beast_executable_linux
else:
    raise Exception("BEAST2 executable not found.")
command = [beast_executable, "-seed", "1", "-overwrite", simulation_xml]
```
This way it should work on both mac and linux.

Reply:
Yes, good point, I'll implement that